packages/pluggableWidgets/datagrid-web/src/components/__tests__/__snapshots__/Table.spec.tsx.snap

```diff
@@ -390,6 +392,7 @@ exports[`Table renders the structure correctly 1`] = `
                 "getHeaderProps": [Function],
                 "getSortByToggleProps": [Function],
                 "getToggleHiddenProps": [Function],
+                "hidden": false,
```

Comment:
How about switching from snapshot tests for this component? Snapshots are so huge now that it doesn't feel realistic to be able to review them

Reply:
The filters, contents and column capabiities are not available while shallowing because it renders inside a renderFunction